### PR TITLE
We need fix in plone.app.z3cform 

### LINKED ISSUE #10: ValueError 'Fieldset specified, but not found' on running from @@mr.migrator view
Note: this was first raised in https://github.com/thet/programmatic.transmogrifier/issues/2, but we now know the fix should be here.

    2015-10-27 13:04:40 INFO Zope Ready to handle requests
    ['', 'jsonsource', 'typemapper', 'removeid', 'pathfixer', 'disable_versioning', 'constructor', 'schemaupdater', 'uid', 'browser_default', 'latlngupdater', 'local_roles', 'workflowupdater', 'owner', 'enable_versioning', 'reindexobject', 'logger']
    ('path', '/opt/bda/content_plone_2015-05-25-13-05-16', <zope.schema._bootstrapfields.TextLine object at 0x7f84fc396f50>, u'/opt/bda/content_plone_2015-05-25-13-05-16')
    ('ignore_path', 'lineage_registry', <zope.schema._bootstrapfields.TextLine object at 0x972f610>, u'lineage_registry')
    ('delete', 'id', <zope.schema._bootstrapfields.TextLine object at 0x972f590>, u'id')
    ('stripstring', '/plone/', <zope.schema._bootstrapfields.TextLine object at 0x9730750>, u'/plone/')
    ('path-key', '_path', <zope.schema._bootstrapfields.TextLine object at 0x9730450>, u'_path')
    ('prependstring', '', <zope.schema._bootstrapfields.TextLine object at 0x9730610>, u'')
    ('level', 'INFO', <zope.schema._bootstrapfields.TextLine object at 0x7393090>, u'INFO')
    ('name', 'PATH', <zope.schema._bootstrapfields.TextLine object at 0x7393c10>, u'PATH')
    ('key', '_path', <zope.schema._bootstrapfields.TextLine object at 0x827a510>, u'_path')
    2015-10-27 13:05:20 ERROR Zope.SiteErrorLog 1445951120.850.677536965988 http://127.0.0.1:8082/plone4/@@mr.migrator-run/@@z3cform_validate_field
    Traceback (innermost last):
      Module ZPublisher.Publish, line 138, in publish
      Module ZPublisher.mapply, line 77, in mapply
      Module Products.PDBDebugMode.runcall, line 70, in pdb_runcall
      Module ZPublisher.Publish, line 48, in call_object
      Module plone.app.z3cform.inline_validation, line 46, in __call__
    ValueError: Fieldset specified, but not found.
    > /home/daniel/dev/Plone/Development/buildout-cache/eggs/plone.app.z3cform-0.7.6-py2.7.egg/plone/app/z3cform/inline_validation.py(46)__call__()
    -> raise ValueError('Fieldset specified, but not found.')

fset = 'none'

### PR DESCRIPTION
**Pull request by djowett**
to avoid ValueError'Fieldset specified, but not found'. Resolves #10

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
     'collective.transmogrifier',
     'setuptools',
     'z3c.autoinclude',
-    'zc.recipe.egg'
+    'zc.recipe.egg',
+    'plone.app.z3cform >=0.7.7,!=1.0' # handle group.__name__ being None
```
Comment (thet) [inline setup.py]: hm... that would be able to pull anything > 1.0, not?
i'd say, it should be 'plone.app.z3cform >=0.7.7,<=1.0'
but then, i can't install plone.app.widgets.

**Comment by thet** [comment]
i think, we shouldn't make this version specifier, but solve it differently. backporting is easier!
AFAIK, we need this commit in the 0.x series:
https://github.com/plone/plone.app.z3cform/commit/045adaadaf9dc491fb7f1f4a22a3675aef095a60

a ``git cherry-pick 045adaadaf9dc491fb7f1f4a22a3675aef095a60`` could be all we need. then test it in buildout.coredev 4.3 branch and if the tests go green, ping eric to release a new 0.x series release. until then use the source checkout.

sounds good?

**Comment by djowett** [comment]
Hi, I think you missed a message in the programmatic.migrator issue...

Basically backporting is already done, and is in version 0.7.7.  

So ... any version of plone.app.z3cform >=0.7.7 will fix this for us, *except* 1.0. Hence the complex pinning.

**Comment by thet** [comment]
ah, got it! know i understand...
cool, let's then merge it!

**Pull request merged**